.github/flaky_test_detection.py

```diff
@@ -0,0 +1,114 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+import requests
+from github import Github
+from github import Auth
+
+
+GIT_ORG = "apache"
+GIT_REPO = "beam"
+GRAFANA_URL = "http://metrics.beam.apache.org"
+#
+ALERT_NAME = "flaky_test"
+READ_ONLY = os.environ.get("READ_ONLY", "false")
+
+
+class Alert:
+    def __init__(self, id, url, name, file_name):
+        self.workflow_id = id
+        self.workflow_url = url
+        self.workflow_name = name
```

Comment:
Based on the usage [here](https://github.com/apache/beam/pull/30345/files#diff-84bdef731e50d6654cfb98a6328e0b480a8c248fbcb97a8d225405ce9b3a3f6bR83), I think it should be renamed to `name` to reflect that it is the name of the alert and not the workflow

Reply:
Correct. Fixed and retested

![image](https://github.com/apache/beam/assets/20559691/c0a0f846-fea1-4f01-bd5a-3edae999ac7d)